tigr.h

```diff
@@ -282,6 +282,9 @@ typedef struct {
 // Returns number of touch points read.
 int tigrTouch(Tigr *bmp, TigrTouchPoint* points, int maxPoints);
 
+// Returns mouse wheel value for a window.

```

Comment:
We should be more specific about what is returned, and make sure that the value returned is comparable in delta / scale on all platforms. I can look into the OSX part.